core/src/main/scala/cats/package.scala

```diff
@@ -31,11 +31,22 @@ package object cats {
    * type `A` to get a pure value of type `B`. That is, the instance
    * encodes pure unary function application.
    */
-  type Id[A] = A
+  type Id[+A] = A
+
+  // Workaround for a compiler bug that should be fixed soon.
```

Comment:
now that the PR on scala is in, can we refer to it here? 
https://github.com/scala/scala/pull/8651

Reply:
@kailuowang Yes, good idea (I linked it in the PR description but the comment would be more helpful).